typescript/cli/src/rebalancer/monitor/Monitor.ts

```diff
@@ -1,19 +1,37 @@
 import EventEmitter from 'events';
 
 import { IRegistry } from '@hyperlane-xyz/registry';
-import { MultiProtocolProvider } from '@hyperlane-xyz/sdk';
-import { WarpCore } from '@hyperlane-xyz/sdk';
-import { objMap, objMerge } from '@hyperlane-xyz/utils';
+import { MultiProtocolProvider, WarpCore } from '@hyperlane-xyz/sdk';
+import { objMap, objMerge, sleep } from '@hyperlane-xyz/utils';
 
 import { IMonitor, MonitorEvent } from '../interfaces/IMonitor.js';
 
+class WrappedError extends Error {
```

Comment:
This could be moved to the utils package or somewhere if we find it useful for other purposes